### ISSUE TITLE
v.4.1.1 throws errors

### ISSUE THREAD

**Issue by vielhuber** [joeyates/imap-backup#108]
ruby -v

```sh
ruby 3.1.0p0 (2021-12-25 revision fb4df44d16) [x86_64-linux]
```

imap-backup status

```sh
/usr/local/rvm/gems/ruby-3.1.0/gems/imap-backup-4.1.1/lib/imap/backup/cli/status.rb:16:in `block in run': undefined method `username' for #<Imap::Backup::Account::Connection:0x00007f8e32e2bcc0 @account=#<Imap::Backup::Account:0x00007f8e32e2c238 @username="XXX", @password="XXX", @local_path="/root/imap-backup/mail/XXX", @folders=[], @server="XXX", @connection_options=nil, @changes={}, @marked_for_deletion=false>, @backup_folders=nil, @client=nil, @config=nil, @folder_names=nil, @provider=nil, @server=nil> (NoMethodError)
```

**Comment by joeyates** [comment]
Hi @vielhuber 

I pushed a fix for this: version 4.1.2

**Comment by joeyates** [comment]
Hi @vielhuber 

I pushed a fix for this: version 4.1.2
